outputs_vnet.tf

```diff
@@ -0,0 +1,36 @@
+output "name" {
+  description = "The resource name of the virtual network."
+  value       = module.avm_res_network_virtualnetwork.name
+}
+
+output "resource_guid" {
```

Comment:
Do we need both of these? For peering you only need the resource id.

What is the difference between these two outputs?